app/layout.tsx

```diff
@@ -9,19 +14,19 @@ import "./styles/globals.css";
 
 import styles from "./layout.module.scss";
 
-export const metadata: Metadata = {
-  title: "Sunrise studio",
-  description: "Shaping the future with digital innovation",
-};
-
 export default function RootLayout({
   children,
 }: {
   children: React.ReactNode;
 }) {
+  const [activeBurger, setActiveBurger] = useState(false);
+  const metadata = globalMetadata;
```

Comment:
I guess this operation is not needed if you're not doing any mutations

Reply:
Done